apps/ewallet_db/lib/ewallet_db/invite.ex

```diff
@@ -93,6 +115,22 @@ defmodule EWalletDB.Invite do
     |> Repo.insert()
   end
 
+  @doc """
+  Accepts an invitation without setting a new password.
+  """
+  @spec accept(%Invite{}) :: {:ok, struct()} | {:error, any()}
+  def accept(invite) do
+    invite = Repo.preload(invite, :user)
+
+    case User.update_without_password(invite.user, %{invite_uuid: nil}) do
+      {:ok, _user} ->
+        Repo.delete(invite)
```

Comment:
Do we actually delete invites or is it soft-delete? Can't we just set them to `used` or something and keep them?

Reply:
It’s hard delete right now. To do soft delete, I'll do this:

1. Disassociate the invite from the user (`user.invite_uuid`) after verification. So `user.invite_uuid == nil`.
2. Add a new `invite.user_uuid` column to keep the user->invite trace after the disassociation from `user.invite_uuid`.
3. Add a new `invite.verified_at` column so it’s easy to see if and when the invite was used.